### ISSUE TITLE
Grouped pattern-matching callbacks not triggering Input or Trigger

### ISSUE THREAD

**Issue by gedemagt** [thedirtyfew/dash-extensions#27]
When using a grouped callback, a Trigger or Input with a pattern-matching id does not trigger the callback. The following example code does not invoke the callback unless I remove the `group="group"`

```python
from dash_extensions.enrich import Dash
from flask import Flask
import dash_html_components as html

from dash.dependencies import ALL
from dash_extensions.enrich import Trigger, Input

fapp = Flask(__name__)
app = Dash(__name__, server=fapp)
app.suppress_callback_exceptions = True

@app.callback(
    Input(dict(type="delete", id=ALL), "n_clicks"), group="group"
)
def do(d):
    print("Triggered")


app.layout = html.Div([
    html.Button("B1", id=dict(type="delete", id="1")),
    html.Button("B2", id=dict(type="delete", id="2"))
])

app.run_server(debug=True)
```

Also - solid work with this awesome package! :+1:

**Comment by emilhe** [comment]
Thanks! I have added experimental support for wildcards with the `group` keywords in the latest rc release. Try it out and let me know if it works as intended,

https://pypi.org/project/dash-extensions/0.0.33rc1/

**Comment by gedemagt** [comment]
Works like a charm!

**Comment by emilhe** [comment]
Great! I have merged the feature into the 0.0.33 release, which is now available on pypi.
